grudge/symbolic/operators.py

```diff
@@ -651,20 +654,32 @@ def matrix(self, afgrp, volgrp, dtype):
                     afgrp.nunit_dofs),
                 dtype=dtype)
 
-        from modepy.tools import UNIT_VERTICES
         import modepy as mp
+
+        vol_basis = volgrp.basis_obj()
+        faces = mp.faces_for_shape(volgrp.shape)
+
+        # NOTE: Assumes each face has the same quadrature/basis
+        face_quadrature = mp.quadrature_for_space(
+            # FIXME: How do we want to handle this in general?
+            # Do we always want a quadrature rule that exactly
+            # integrates 2N polynomials on the faces?
+            # For tensor-product and simplices?
```

Comment:
Why not? What downside do you see?

Reply:
I was thinking about over integration, but I think my latest commit addresses this issue.